docs/tutorials/Manual-Signature-Verification/Readme.md

```diff
@@ -0,0 +1,268 @@
+
+You can use the following RESTful API for that:
+
+#### POST /rest/v3/signature/verify
+
+```json
+{
+  "requestObject": {
+    "activationId": "string",
+    "applicationKey": "string",
+    "data": "string",
+    "signature": "string",
+    "signatureType": "ENUM_SIGNATURE_TYPE",
+    "signatureVersion": "string"
+  }
+}
+```
+
+**Request Body Structure**
+
+| Parameter | Description | Obtained at |
+|---|---|---|
+| `activationId` | ID of the activation. | Comes in the signature HTTP header. |
+| `applicationKey` | Key specific for an application version. | Comes in the signature HTTP header. |
+| `data` | Signature base string. | We computed this value during the process. |
+| `signature` | Signature value. | Comes in the signature HTTP header. |
+| `signatureType` | Signature type. | Comes in the signature HTTP header. Note: The PowerAuth Service uses enum values in upper-case, while the HTTP header comes in lower-case format. The values are otherwise the same. |
+| `signatureVersion` | Signature version. | Comes in the signature HTTP header. |
+
+You will receive the following response:
+
+```json
+{
+  "status": "string",
+  "responseObject": {
+    "signatureValid": true,
+    "activationId": "string",
+    "activationStatus": "ENUM_ACTIVATION_STATUS",
+    "userId": "string",
+    "applicationId": 0,
+    "blockedReason": "string",
+    "remainingAttempts": 0,
+    "signatureType": "ENUM_SIGNATURE_TYPE"
+  }
+}
+```
+
+**Response Body Structure**
+
+| Parameter | Description |
+|---|---|
+| `status` | Status of the request processing, `OK` if the HTTP request executes correctly, `ERROR` if an error occurs. |
+| `responseObject.signatureValid` | Boolean indicating if signature was verified correctly or if the signature was invalid. |
+| `responseObject.activationId` | Activation ID of the activation used to verify the signature. |
+| `responseObject.activationStatus` | Status of the activation during the verification attempt (`ACTIVE`, `BLOCKED`, `REMOVED`, unless you made an implementation error, you should not obtain `CREATED` or `PENDING_COMMIT` values.) |
+| `responseObject.userId` | User ID of the user who is associated with this activation. |
+| `responseObject.applicationId` | Application ID of an application that was used to compute the signature. |
+| `responseObject.blockedReason` | In case the activation is blocked, this value provides more information about the cause. It can be any value provided while calling the service that blocks the activation, or a `MAX_FAILED_ATTEMPTS` value in case the activation was blocked due to too many authentication failed attempts. |
+| `responseObject.remainingAttempts` | Number of the remaining authentication attempts. |
+| `responseObject.signatureType` | Used signature type. |
+
+_Note: We also have a SOAP service. If you like this better, [here is the documentation](https://developers.wultra.com/docs/2020.05/powerauth-server/SOAP-Service-Methods), including the link to WSDL definitions._
```

Comment:
I suggest not to use version `2020.05`, the documentation may become obsolete.

Reply:
Yes, correct.